components/brave_wallet/browser/brave_wallet_constants.cc

```diff
@@ -4,6 +4,7 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include <map>
+#include <set>
```

Comment:
Sorry, this is an unused code refuse from a previous commit where I tried a solution using sets. Solution that I reverted because it was too problematic.

Reply:
^ cc @simoarpe this hasn't been removed yet